thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/AnomalyUtils.java

```diff
@@ -117,7 +118,7 @@ public MetaDataNode(MergedAnomalyResultDTO anomaly){
       this.startTimeISO = new Timestamp(anomaly.getStartTime()).toString();
       this.endTimeISO = new Timestamp(anomaly.getEndTime()).toString();
       this.functionName = anomaly.getFunction().getFunctionName();
-      this.feedback = anomaly.getFeedback();
+      this.feedback = (anomaly.getFeedback() == null)? "null" : String.valueOf(anomaly.getFeedback().getFeedbackType());
```

Comment:
There seems a gap between Feedback and FeedbackType. For feedbackType there is "NO_FEEDBACK", but if anomaly.getFeedback() is null, it then cannot access .getFeedbackType() thus as always we cannot utilize "NO_FEEDBACK" as an anomaly feedback?